Interview Questions/Questions.md

```diff
@@ -32,3 +32,17 @@ public static void main(String[] args)
     //output will be 10 20
 }
 ```
+
+###Q3- Can we declare the main method of our class as private?
+
+A3- 
+In java, main method must be public static in order to run any application correctly. If main method is declared as private, developer won't get any compilation error however, it will not get executed and will give a runtime error. 
+
+###Q4- Is there any way to skip Finally block of exception even if some exception occurs in the exception block?
+
```

Comment:
Hey, can you please add `A4-` to it, lets keep it consistent. 

Thank you for your contribution.